stable/admin/values.yaml

```diff
@@ -126,6 +126,48 @@ admin:
   #   type: LoadBalancer
   #   annotations: {}
 
+  # Enable NuoDB external access via Ingress; the provisioned Ingress controller
+  # should support SSL passthrough feature allowing it to send the TLS
+  # connections directly to the NuoDB Admin instead of decrypting the
+  # communication; supported starting with NuoDB image version 4.2.3
+  ingress:
+    enabled: false
+    api:
+      # the fully qualified domain name (FQDN) of the network host for the NuoDB
+      # Admin REST API
+      hostname: ""
+      # path that is matched against the path of the incoming HTTP request
+      path: /
+      # the associated IngressClass name defines which controller will implement
+      # the resource; for Kubernetes >= 1.18 the ingress class name should be
+      # specified via the className option instead of using annotation
+      className: ""
+      # custom annotations that are set on the Ingress resource
+      annotations: 
+        ingress.kubernetes.io/ssl-passthrough: "true"
+      # Enable TLS termination in the ingress controller; it is recommended to
+      # use SSL Passthrough feature instead
+      tls: false
+      # the name of the secret used by Ingress controller to terminate the TLS
+      # traffic on port 443
+      secretName: ""
+    sql:
+      # the fully qualified domain name (FQDN) of the network host for SQL
+      # clients
+      hostname: ""
+      # For Kubernetes >= 1.18 the ingress class name should be specified via the
+      # ingressClassName option instead of using annotation
+      className: ""
+      # custom annotations that are set on the Ingress resource
+      annotations: 
+        ingress.kubernetes.io/ssl-passthrough: "true"
+      # Enable TLS termination in the ingress controller; it is recommended to
+      # use SSL Passthrough feature instead
+      tls: false
+      # the name of the secret used by Ingress controller to terminate the TLS
+      # traffic on port 443
+      secretName: ""
```

Comment:
How would this work? Would the Ingress controller communicate over plaintext TCP with the admin?

Reply:
Using a secret to terminate the SQL client TLS connection on the HAProxy with `ssl-passthrough=false` doesn't work nowadays probably because the ingress controller assumes that the traffic is HTTP and tries to inspect the headers, etc. 

I didn't test with different ingress controllers so I decided to keep these values to allow maximum configuration flexibility and support of a variety of use cases in the future.

Reply:
My preference would be to only expose configuration options that we know work. The fact that we are exposing these options and providing some documentation implies that we have validated somehow, but we only know that they definitely _don't_ work with HAProxy.

Reply:
Okay, I was having the same dilemma. Making it flexible in terms of configuration or exposing only those options that we have tested with. We will assume that the SSL passthough feature is turned `on` don't expose the following variables at all:
* `admin.ingress.api.path`
* `admin.ingress.api.tls`
* `admin.ingress.api.secretName`
* `admin.ingress.sql.tls`
* `admin.ingress.sql.secretName`
* `database.te.ingress.tls`
* `database.te.ingress.secretName`